src/java/main/org/apache/zookeeper/client/ConnectStringParser.java

```diff
@@ -20,8 +20,11 @@
 
 import java.net.InetSocketAddress;
 import java.util.ArrayList;
+import java.util.List;
 
 import org.apache.zookeeper.common.PathUtils;
+import static org.apache.zookeeper.common.StringUtils.split;
```

Comment:
nit: I could be wrong on this, but I think in general we prefer to import classes and not static methods directly.

Reply:
eribeiro: I have kept the code same, in master and branch-3.5 static method member is imported i.e import static org.apache.zookeeper.common.StringUtils.split

Reply:
eribeiro: Can you please complete the review comments? Can you please approve the changes so that patch can be created.